portfolio/src/main/webapp/script.js

```diff
@@ -27,12 +27,12 @@ function addRandomFact() {
   factContainer.innerText = fact;
 }
 
-function fetchMessage() {
+function fetchComments() {
     fetch("/data").then(request => request.json()).then((textResponse) => {
-        for (var i = 0; i < 3; i++) {
+        for (var i = 0; i < textResponse.length; i++) {
             var paragraph_tag = document.createElement("p");
```

Comment:
Nit: Let's use camelcase for paragraph_tag to maintain consistency in variable naming.

Reply:
Updated variable name.